sklearn/model_selection/_split.py

```diff
@@ -1971,6 +1967,12 @@ def train_test_split(*arrays, **options):
             If the input is sparse, the output will be a
             ``scipy.sparse.csr_matrix``. Else, output type is the same as the
             input type.
+            
+    Notes
+    -----
+    Randomized CV splitters may return different results for each call of
+    split. You can make the results identical by setting ``random_state``
```

Comment:
Not applicable here: this is not a sitter with a `split` method

Reply:
Good catch! Thank you. I will remove this update